### PR TITLE
LPS-205933 Use the attribute model name if sent

### PR DESCRIPTION
**Pull request by 4lejandrito**
From https://github.com/liferay-headless/liferay-portal/pull/1812:

Steps to reproduce in https://liferay.atlassian.net/browse/LPS-205933

In this case the problem seems related with the current use of the method parameters in the export-import API:

In com.liferay.document.library.internal.exportimport.data.handler.FolderStagedModelDataHandler#doImportStagedModel we are doing this:

```
portletDataContext.importClassedModel(
			folder, importedFolder, DLFolder.class);
```

The third parameter in other cases (importLocks or importPermissions) is been used to set the right class name, but in the case of _importWorkflowDefinitionLink it is been extracted from the classed model.

I've changed this and use this className as when sent, and it fixes the issue.

### CODE REVIEW

**Comment by 4lejandrito** [comment]
@magjed4289 please review the test, @vendeltoreki review the PR.

**Comment by 4lejandrito** [comment]
ci:test:playwright

**Comment by 4lejandrito** [comment]
@jaimelr10 I added a playwright test to reproduce this bug, you might be interested.

File: modules/test/playwright/pages/document-library-web/documentLibrary.page.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	async editEntry(entryTitle: string) {
+		await this.page
+			.locator(`.card-body:has-text('${entryTitle}')`)
+			.getByLabel('More actions')
+			.click();
+		await this.page.getByRole('menuitem', {name: 'Edit'}).click();
+	}
+
+	async openOptionsMenu() {
+		await this.optionsMenu.click();
```
Comment (4lejandrito) [inline modules/test/playwright/pages/document-library-web/documentLibrary.page.ts:40]: This is flaky! Sometimes the JS has not loaded yet when clicking, and clicking does nothing.

Comment (4lejandrito) [reply modules/test/playwright/pages/document-library-web/documentLibrary.page.ts:40]: This fixes it:
```ts
async openOptionsMenu() {
  this.page.waitForTimeout(1000)
  await this.optionsMenu.click();
}
```

File: modules/test/playwright/tests/export-import-web/import.spec.ts [hunk 2]
```diff
@@ -5,60 +5,29 @@
...
 	expect(
-		await page
-			.getByTitle('Workflow Definition')
-			.evaluate(
-				(select: HTMLSelectElement) =>
-					select.options[select.selectedIndex].value
-			)
+		await _documentLibraryEditFolderPage.getSelectedWorkflowDefinition()
 	).toBe('Single Approver@1');
 });
```
Comment (4lejandrito) [inline modules/test/playwright/tests/export-import-web/import.spec.ts:33]: There is no teardown! this test leaves garbage.

Comment (4lejandrito) [reply modules/test/playwright/tests/export-import-web/import.spec.ts:43]: Fixed by https://github.com/liferay-headless/liferay-portal/pull/1816/commits/5486549b502002632ff1440dbcaa62a13cebd9c2.

**Review by vendeltoreki — APPROVED** [review]
The code changes look good to me. We should run the `staging` CI test package as well, just to be sure.

**Comment by vendeltoreki** [comment]
ci:test:staging

**Comment by magjed4289** [comment]
ci:test:playwright

**Review by magjed4289 — APPROVED** [review]
Just a little change added as some of local test executions were failing due to elements not being loaded correctly. The rest of the test looks good!

File: modules/test/playwright/pages/document-library-web/documentLibrary.page.ts (hunk 1 above)
Comment (4lejandrito) [reply modules/test/playwright/pages/document-library-web/documentLibrary.page.ts:41]: Fixed by `await this.page.waitForLoadState('networkidle');`.

**Comment by 4lejandrito** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:test:staging

**Comment by vendeltoreki** [comment]
Staging CI failures don't seem to be relevant, so it's good to go from that side.

**Comment by 4lejandrito** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/146290

**Comment by 4lejandrito** [comment]
https://github.com/brianchandotcom/liferay-portal/pull/146291

**Pull request closed without merging**